test1.as

```diff
@@ -1,12 +1,10 @@
-.entry LOOP
```

Comment:
revert this test to the prev test.

fainely need to produce our tests for our project

Reply:
not moving for now.
done